app/src/main/java/io/homeassistant/companion/android/PresenterModule.kt

```diff
@@ -122,5 +123,8 @@ class PresenterModule {
 
         @Binds
         fun bindLockPresenter(presenter: LockPresenterImpl): LockPresenter
+
+        @Binds
+        fun bindSensorUpdater(sensorUpdater: AllSensorsUpdaterImpl): SensorUpdater
```

Comment:
I noticed you never actually inject this anywhere, if that's the case there is no reason to add it here.  If you want to use dagger make sure you always have it injected rather than create it every use.

Reply:
Ugh yeah this was a relic from when I was putting it into the Dagger graph. The PR got a little large so I backed up from that. Will remove.

Reply:
Done